README.md

```diff
@@ -75,33 +16,47 @@ In order to build the project, we need to run the following command:
 
```

Comment:
I think you shouldn't have had these problems if you execute `npm install` from the start before running any other command. 

I have updated the README because it was not specified correctly that it is necessary to execute `npm install`  before `npm run build.`

Reply:
Yes! Worked fine! Thanks!